src/main/PDF/Services/CrudServices/ImageToPDFService.java

```diff
@@ -0,0 +1,102 @@
+package PDF.Services.CrudServices;
+
+import Config.Message;
+import Config.Service;
+import PDF.PdfMessage;
+import org.apache.commons.io.output.ByteArrayOutputStream;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.graphics.image.LosslessFactory;
+import org.apache.pdfbox.pdmodel.graphics.image.PDImageXObject;
+
+import javax.imageio.ImageIO;
+import java.awt.geom.AffineTransform;
+import java.awt.image.AffineTransformOp;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class ImageToPDFService implements Service {
+    public static final int PDF_HEIGHT = 792;
+    public static final int PDF_WIDTH = 612;
+
+    InputStream fileStream;
```

Comment:
Can you make this private and create a getFileStream() function

Reply:
Good catch, done